debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/SourceRecordAsserter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.sqlserver.util;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.fest.assertions.Assertions;
+
+public class SourceRecordAsserter {
```

Comment:
Seems that could go to core? And I'd suggest to name it `SourceRecordAsserter` to be consistent with FEST Assert's convention.

Reply:
You mean `SourceRecordAssert`?

Reply:
Does it make sense for you to keep `SourceRecordAssert` when we have `VerifyRecord`?

My intention was to build an asserter which hides chain of getter calls and casts to Struct. On the other hand, aren't the two methods proposed below too "detailed", are they?

Reply:
So yes, I generally like the `assertThat()` approach. We'll have to see though how to consolidate that with the existing tests.